employee-rostering-distribution/src/main/assembly/assembly-optaweb-employee-rostering.xml

```diff
@@ -45,6 +44,14 @@
   </fileSets>
 
   <dependencySets>
+    <dependencySet>
+      <includes>
+        <include>org.optaweb.employeerostering:employee-rostering-frontend:jar</include>
+      </includes>
+      <outputDirectory>binaries</outputDirectory>
+      <useStrictFiltering>true</useStrictFiltering>
+      <useTransitiveFiltering>false</useTransitiveFiltering><!-- Only include war -->
```

Comment:
I guess the comment on the right is just from some copy&paste line

Reply:
@marekn from the previous version